lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -256,6 +256,8 @@ def inset_axes(parent_axes, width, height, loc='upper right',
                axes_class=None, axes_kwargs=None,
                borderpad=0.5):
     """
+    versionchanged:: 3.11
+    The *borderpad* parameter now accepts a tuple of (x, y) paddings.
```

Comment:
I think this should be `versionadded` because you are introducing a new option but the old option still works.  Also it should go under the parameter description.  For a similar example see

https://github.com/matplotlib/matplotlib/blob/011d12f1d6f67f1ae66cb8860b323cc75409d26b/lib/matplotlib/stackplot.py#L67-L68